npm-shrinkwrap.json

```diff
@@ -47,26 +50,43 @@
     "ast-types": {
       "version": "0.8.15"
     },
+    "async-lock": {
+      "version": "1.0.0"
+    },
     "asynckit": {
       "version": "0.4.0"
     },
     "aws-sign2": {
       "version": "0.6.0"
     },
     "aws4": {
-      "version": "1.5.0"
-    },
-    "babel-code-frame": {
-      "version": "6.16.0"
+      "version": "1.6.0"
     },
     "babel-messages": {
-      "version": "6.8.0"
+      "version": "6.23.0",
```

Comment:
Thanks for the instructions, Bartek. I think I've cleaned this up by reverting the dep changes, installed new deps specifically `npm install --save async-lock && npm install --save raven@2.1.1` and finally `npm run shonkwrap`.